peachjam/models/judgment.py

```diff
@@ -441,6 +445,22 @@ def pre_save(self):
             self.assign_title()
         super().pre_save()
 
+    def convert_html_to_pdf(self):
+        with NamedTemporaryFile(suffix=".html") as html_file:
+            html_file.write(self.content_html.encode("utf-8"))
```

Comment:
I suggest you do something like `html_file.write(self.prepare_content_html_for_pdf())` so that subclasses can override the HTML that is used to generate the PDF, such as by adding a logo at the top.

Also, I think this can safely all be done on the CoreDocument object.

Reply:
ok, done